File: app/src/main/java/com/plbear/iweight/Utils/Constant.java
package com.plbear.iweight.Utils;

import android.net.Uri;

/**
 * Created by yanyongjun on 2017/5/19.
 */

public class Constant {
    public static final String SP_NAME = "iweight";

    public static final String PROVIDER_AUTHORITY = "com.plbear.iweight.provider";
    public static final int PROVIDER_ITEM = 1;
    public static final int PROVIDER_ITEMS_WITHOUT_NOTIFY = 2;
    public static final int PROVIDER_POS = 3;

    public static final Uri CONTENT_URI = Uri.parse("content://"+PROVIDER_AUTHORITY+"/items");
    public static final Uri CONTENT_URI_WITHOUT_NOTIRY = Uri.parse("content://"+PROVIDER_AUTHORITY+"/items_without_notify");

}

File: app/src/main/java/com/plbear/iweight/utils/Constant.kt
package com.plbear.iweight.utils

import android.net.Uri

/**
 * Created by yanyongjun on 2018/1/28.
 */
class Constant {
    companion object {
        val SP_NAME = "iweight"

        var PROVIDER_AUTHORITY = "com.plbear.iweight.provider"
        var PROVIDER_ITEM = 1
        var PROVIDER_ITEMS_WITHOUT_NOTIFY = 2
        var PROVIDER_POS = 3

        var CONTENT_URI = Uri.parse("content://$PROVIDER_AUTHORITY/items")
        var CONTENT_URI_WITHOUT_NOTIRY = Uri.parse("content://$PROVIDER_AUTHORITY/items_without_notify")
    }
}